helm/csi-powerstore/templates/_helpers.tpl

```diff
@@ -47,4 +47,15 @@ Return the appropriate sidecar images based on k8s version
       {{- print "gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.6.0" -}}
     {{- end -}}
   {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+ 
+{{/*
+Return true if storage capacity tracking is enabled and is supported based on k8s version
+*/}}
+{{- define "csi-powerstore.isStorageCapacitySupported" -}}
+{{- if eq .Values.storageCapacity.enabled true -}}
+  {{- if and (eq .Capabilities.KubeVersion.Major "1") (ge (trimSuffix "+" .Capabilities.KubeVersion.Minor) "24") -}}
```

Comment:
Since we are going to support 1.25 k8 version, so do we need to update the check here too?

Reply:
Capacity tracking's GA is from k8 v1.24. This template is added to enable the feature only for k8 versions >= 1.24, otherwise it will be disabled.